Riot/Modules/CreateRoom/EnterNewRoomDetails/EnterNewRoomDetailsViewController.swift

```diff
@@ -280,7 +280,7 @@ final class EnterNewRoomDetailsViewController: UIViewController {
         super.viewWillAppear(animated)
         
         self.keyboardAvoider?.startAvoiding()
-        screenTracker.trackScreen()
+//        screenTracker.trackScreen()
```

Comment:
I'm not sure we have to comment this screenTracker in Tchap, the corresponding code should be disabled thanks to BuildSettings.analyticsConfiguration where `isEnabled =false`